data/modules/scripts/gamestore/init.lua

```diff
@@ -1637,16 +1637,18 @@ function GameStore.processHouseRelatedPurchase(player, offer)
 	end
 	if inbox then
 		for _, itemId in ipairs(itemIds) do
-			local decoKit = inbox:addItem(ITEM_DECORATION_KIT, 1)
-			if decoKit then
-				decoKit:setAttribute(ITEM_ATTRIBUTE_DESCRIPTION, "You bought this item in the Store.\nUnwrap it in your own house to create a <" .. ItemType(itemId):getName() .. ">.")
-				decoKit:setCustomAttribute("unWrapId", itemId)
-				if isCaskItem(itemId) then
-					decoKit:setAttribute(ITEM_ATTRIBUTE_DATE, offer.count)
-				end
+			for i = 1, offer.count do
+				local decoKit = inbox:addItem(ITEM_DECORATION_KIT, 1)
+				if decoKit then
+					decoKit:setAttribute(ITEM_ATTRIBUTE_DESCRIPTION, "You bought this item in the Store.\nUnwrap it in your own house to create a <" .. ItemType(itemId):getName() .. ">.")
+					decoKit:setCustomAttribute("unWrapId", itemId)
+					if isCaskItem(itemId) then
+						decoKit:setAttribute(ITEM_ATTRIBUTE_DATE, offer.count)
+					end
 
-				if offer.movable ~= true then
-					decoKit:setAttribute(ITEM_ATTRIBUTE_STORE, systemTime())
+					if offer.moveable ~= true then
```

Comment:
var moveable doesn't exists anymore.
```suggestion
					if offer.movable ~= true then
```